modules/akmods/module.yml

```
@@ -3,6 +3,7 @@ shortdesc: The akmods module is a tool used for managing and installing kernel m
 readme: https://raw.githubusercontent.com/blue-build/modules/main/modules/akmods/README.md
 example: |
   type: akmods
+  base: asus # if not specified, classic "main" base is used by default
```

```suggestion
  type: akmods
  base: asus # if not specified, classic "main" base is used by default
  nvidia-version: 545 # Will add rpms from the corresponding nvidia image
```

I think we should show all the properties here.